web/src/engine/dom-utils/src/stylesheets.ts

```diff
@@ -106,8 +108,9 @@ export class StylesheetManager {
     // Modern browsers: use WOFF, TTF and fallback finally to SVG. Don't provide EOT
     if(os == DeviceSpec.OperatingSystem.iOS) {
       if(ttf != '') {
-        // once upon a time, we did a cache-prevention thing here... but that was removed long, long ago.
-        // ttf = this.unCached(ttf);
```

Comment:
Good thing I checked - I was wrong!  Fortunately, the `kmwembedded.ts` part I'd missed had not been removed yet, so I caught this mistake and remedied it here.